app/models/manageiq/providers/kubevirt/infra_manager/provision/cloning.rb

```diff
@@ -65,10 +65,7 @@ def start_clone(options)
           template: {
             spec: {
               domain: {
-                memory: {
-                  value: memory.to_i,
-                  unit: 'MiB'
-                }
+                memory: memory.to_s + 'MiB'
```

Comment:
According to the Kubenetes documentation the recommended way to specify the memory in MiB is using the `Mi` suffix, without the `B`. See here for details: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-memory